### ISSUE TITLE
[Feature Request] Option to Include Apostrophe (') in Suggestions

### ISSUE THREAD

**Issue by jlimcode** [tadashi-aikawa/obsidian-various-complements-plugin#307]
In English, conjunctions like `doesn't` are used as one word, but with my current various-complements settings, the suggested completion is `doesn` rather than the complete conjunction. 

I'm not sure if it's helpful for other users to have apostrophes treated as border characters in the tokenizer, but I'd love a setting to toggle single quotes (') as recognized characters in a word. This would also be helpful for suggesting possessives. 

So to recap, I'd like a setting to treat `doesn't` as one word, `doesn't`, rather than `doesn` and `t`. If this setting already exists, please let me know. 

Thank you for creating/maintaining this plugin! It's so useful and I wouldn't enjoy Obsidian without it.

**Comment by tadashi-aikawa** [comment]
Hi, @jlimcode :)

> If this setting already exists, please let me know.

No, it doesn't exist. 

As a workaround, you can register frequently used keywords like "doesn't" in Custom dictionaries.

In Various Complements, single quotes are considered as quotation marks and are excluded from the tokenizer's attention. Simply adding single quotes to the tokenizer's attention would result in noisy candidates. Therefore, it is necessary to exclude only single quotes at the ends of words.

If there are others who need this feature, I would like to consider it. Until then, please set the status to gathering intersects.
